src/App.tsx

```diff
@@ -372,67 +368,30 @@ export const App = ({
   }
 
   return (
-    <div className={containerStyles}>
-      {user && !isClosedForComments && (
-        <CommentForm
-          shortUrl={shortUrl}
-          onAddComment={onAddComment}
-          user={user}
-        />
-      )}
-      {!!picks.length && (
-        <TopPicks
-          baseUrl={baseUrl}
-          pillar={pillar}
-          comments={picks}
-          isSignedIn={!!user}
-        />
-      )}
-      <Filters
-        filters={filters}
-        onFilterChange={onFilterChange}
-        totalPages={totalPages}
-        commentCount={commentCount}
-      />
-      {showPagination && (
-        <Pagination
+    <Column>
+      <>
```

Comment:
This wrapper fragment is needed to prevent typescript complaining that we're pass 'undefined' or 'false' as children

Reply:
I much prefer wrapping in fragments , its easier on the eyes 👍